### PR TITLE
Release notes 7.3.0

### PR DESCRIPTION
**Pull request by yaauie**
Includes a cherry-pick of the 7.2.0 release notes

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-7-3-0,Logstash 7.3.0>>
+* <<logstash-7-2-0,Logstash 7.2.0>>
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: Do we need 7.2.1?

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -13,6 +15,156 @@ This section summarizes the changes in the following releases:
...
+
+*Jms Input*
+
+* The JMS Input is now a default plugin distributed with Logstash
+
+*Snmp Input*
+
+* Adding oid_path_length config option https://github.com/logstash-plugins/logstash-input-snmp/pull/59[#59]
+* Fixing bug with table support removing index value from OIDs https://github.com/logstash-plugins/logstash-input-snmp/issues/60[#60]
+
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: Should this issue be closed?  It looks at least partially resolved with https://github.com/logstash-plugins/logstash-input-snmp/pull/59

Comment (yaauie) [reply docs/static/releasenotes.asciidoc]: It _looks_ like it is fixed with #59, but we are leaving the ticket open pending verification from the person who reported it.

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (yaauie) [reply docs/static/releasenotes.asciidoc]: :thinking: I honestly wouldn't expect any of the patch-level releases to be included in a changelog for a minor-level release, and am a bit surprised to see them listed here. The changelog entry below for 7.3.0 includes fixes that _also_ target a patch-level release on 7.2 (which hasn't been released), so I don't think there is risk in omitting it.

**Pull request merged**